tests/python/test_linalg.py

```diff
@@ -3,6 +3,106 @@
 from taichi import approx
 
 
+@ti.all_archs
+def test_BasicUlts():
```

Comment:
Thank for adding these linalg tests! But note that we use underline code style for function names:
```suggestion
def test_basic_utils():
```

Reply:
sure I will also check others

Reply:
what about vars in the function, should they also use underline?